.github/workflows/create-new-release.yml

```
@@ -0,0 +1,67 @@
+
+      - name: Setup Git
+        run: |
+          git config --global user.name "${{ github.actor }}"
+          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
+
+      - name: Fetch All Branches
+        run: git fetch --all
+
+      - name: Get All Release Branches
+        id: release_branches
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)